cli/man/TEMPLATE.1.md.example

```diff
@@ -0,0 +1,69 @@
+% COMMAND(1) Cargill, Incorporated | Splinter Commands
+
+<!--
+This file should be named the full command (with dashes between subcommands)
```

Comment:
To clarify, I suggest changing to:

"Template for a man page source file in Markdown format. The  file should have the name of the command plus any subcommands (with dashes ...)

Reply:
updated